system_test/common/aest_mempool_SUITE.erl

```diff
@@ -69,7 +69,8 @@
     name    => node2,
     peers   => [node1],
     backend => aest_docker,
-    source  => {pull, "aeternity/aeternity:local"}
+    source  => {pull, "aeternity/aeternity:local"},
+    mining => #{autostart => false}
```

Comment:
This solves the issue with a couple of nodes mining concurently, thus producing forks